elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -176,7 +174,8 @@ class PfePrimaryDetail extends PFElement {
     window.addEventListener("resize", this._debouncedSetBreakpoint);
 
     // Process the light DOM on any update
-    this._observer.observe(this, lightDomObserverConfig);
+    this._childObserver.observe(this, { childList: true });
+    this._attributesObserver.observe(this, { attributes: true, attributeFilter: ["active"] });
```

Comment:
Should we add the `id` attribute to the `attributeFiler`?  Since we are also updating the "id" property internally? 🤔

Reply:
Eh? I'm not using it or updating it on the host, only the nav items?